addons/mail/static/src/core/common/message_model.js

```diff
@@ -358,6 +358,53 @@ export class Message extends Record {
         );
     }
 
+    /**
+     * Checks if the message requires a preview.
+     * A message needs a preview if its body is empty and it has one or more attachments.
+     */
+    get hasOnlyAttachments() {
+        return this.isBodyEmpty && this.attachment_ids.length > 0;
+    }
+
+    get messagePreviewText() {
+        if (!this.hasOnlyAttachments) {
+            return this.inlineBody || this.subtype_description;
+        }
+        const { attachment_ids: attachments } = this;
+        if (!attachments || attachments.length === 0) return "";
+        switch (attachments.length) {
```

Comment:
Inline `if` statement is not accepted by our linter tools. Make sure they are properly installed (see `@web/tooling/enable.sh`)